src/main/java/org/catmaid/ScaleCATMAID.java

```diff
@@ -72,46 +72,55 @@
 	{
 		public int tileWidth;
 		public int tileHeight;
+		public long minC;
```

Comment:
I do not think that this is useful because scaling usually processes an entire z-section from full resolution down until it becomes a single tile or less.  Processing only parts of it does not make sense.  What is the rational behind this?